src/rsz/src/RepairDesign.cc

```diff
@@ -1234,6 +1234,11 @@ RepairDesign::makeRepeater(const char *reason,
     for (const Pin *pin : load_pins) {
       Port *port = network_->port(pin);
       Instance *inst = network_->instance(pin);
+
+      // do not disconnect/reconnect don't touch instances
+      if (resizer_->dontTouch(inst)) {
+        continue;
+}
```

Comment:
Looks like the indenting on this is incorrect.

Reply:
I guess github's web interface does not do correct indentations by default :). Will fix. Thx